core/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -1029,13 +1030,20 @@ class QualificationSuite extends FunSuite with BeforeAndAfterEach with Logging {
     val valuesDetailed = rowsDetailedOut(1).split(",")
     assert(headersDetailed.size == QualOutputWriter
       .getDetailedHeaderStringsAndSizes(Seq(qualApp.aggregateStats.get), false).keys.size)
-    //UnsupportedeExecs and UnsupportedExprs is not present in the file
-    assert(headersDetailed.size -2  == csvDetailedFields.size)
-    assert(valuesDetailed.size - 1 == csvDetailedFields.size) // UnsupportedExprs is empty
+    // UnsupportedeExecs, UnsupportedExprs, and Estimated Job Frequency are present in the file
+    // but not present in csvDetailedFields definition
+    assert(headersDetailed.size - 3 == csvDetailedFields.size)
+    // UnsupportedExprs is empty, but Estimated Job Frequency forces the value to exist
+    assert(valuesDetailed.size - 3 == csvDetailedFields.size)
     // check all headers exists
     for (ind <- 0 until csvDetailedFields.size) {
       assert(csvDetailedHeader(ind).equals(headersDetailed(ind)))
     }
+    // check for estimated frequency column
+    val estimatedFrequencyIndex = 25
+    assert(headersDetailed(estimatedFrequencyIndex).equals("Estimated Job Frequency (monthly)"))
+    assert(valuesDetailed(estimatedFrequencyIndex).toLong == DEFAULT_JOB_FREQUENCY)
```

Comment:
need more tests, this is for default value, you aren't testing it actually does the calculations properly, this is also only testing the running qualification app and not the .log file output.  Did we update results and compare the .log in another test.